app/core.py

```diff
@@ -0,0 +1,47 @@
+"""Main functionalities."""
+
+import app.image.image as image
+import app.image.color_format as color_format
+import app.parser.factory as factory
```

Comment:
For consistency I would change that to relative path and import only used "parts"
i. e. `from .image.image import (Image, RawDataContainer)`